src/chess_piece_move_validation_tests/bishop/bishop.ino

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file bishop.ino
+ * @author Ines Rohrbach, Nico Schramm
+ * @brief Test for ECPBishop isMoveValid method.
+ * @version 0.1
+ * @date 2024-11-27
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#include <Dezibot.h>
+#include <EmbeddedChessPieces.h>
+#include <Wire.h>
+
+#define BAUD_RATE 9600
+
+Dezibot dezibot = Dezibot();
+
+void setup() {
+    Serial.begin(BAUD_RATE);
+    dezibot.begin();
+    delay(500);
+}
+
+void loop() {
+    Serial.println("\n=== STARTING TEST ===\n");
+
+    Serial.println("Testing bishop...");
+    testBishop();
+
+    Serial.println("\n=== TEST DONE ===");
+    Serial.println("Sleeping for 10 seconds...\n");
+
+    delay(10000);
+}
+
+void testBishop() {
+    const ECPChessField initialField = { D, 4 };
+    ECPBishop bishop = ECPBishop(dezibot, initialField, false);
+
+    // valid moves
+    Serial.println("\nTesting valid moves...");
+    ECPChessField validFields[] = {
+                                                                             { H, 8 },
+        { A, 7 },                                                   { G, 7 },
+                  { B, 6 },                               { F, 6 },
+                            { C, 5 },           { E, 5 },
+                                      /* fig */
+                  { B, 3 }, { C, 3 },           { E, 3 },
+                  { B, 2 },                               { F, 2 },
+        { A, 1 },                                                   { H, 1 }
+    };
```

Comment:
Test (correctly) fails for { B, 3 } and { H, 1 }.
Please correct the valid fields.

And move `};` to the line above for consistency.

Reply:
Fixed in 996f4ab15645c8fc9510b49b5a6d509155402c88